samples/simple/README.rst

```diff
@@ -0,0 +1,94 @@
+.. _simple_sample:
+
+Simple
+######
+
+.. contents::
+   :local:
+   :depth: 2
+
+The simple sample demonstrates how to setup CHARGER, LEDs and USB CC events with npmx drivers.
```

Comment:
```suggestion
The Simple sample demonstrates how to perform basic activities on CHARGER, LEDs and USB CC events with npmx drivers.
```

Reply:
Additionally, in this sentence we mentioned 3 of 4 activities from Overview section - should the fourth one be mentioned somehow?

Reply:
Added information about ADC.